### PR TITLE
SOC-1370 Discussion splash pages for 3 more communities

### PR DESCRIPTION
**Pull request by bkoval**
https://wikia-inc.atlassian.net/browse/SOC-1370

@garthwebb @rafalkalinski

### CODE REVIEW

**Comment by bkoval** [comment]
Let me confirm the iOS links with the app team, they're a bit different for the new communities.

**Comment by bkoval** [comment]
iOS links unified :)

File: front/scripts/auth/signup/SignupForm.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ class SignupForm {
 
 	private getFormValues(): HeliosRegisterInput {
 		var formElements: HTMLCollection = this.form.elements;
+
```
Comment (bkoval) [inline front/scripts/auth/signup/SignupForm.ts]: I needed to :)

**Comment by rafalkalinski** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**